tests/unit/main_html5/wrapper-api-tests.js

```diff
@@ -1126,6 +1126,16 @@ describe('main_html5 wrapper tests', function () {
     ]);
   });
 
+  it('setAudio should set the audio by id and return true', function(){
+    element.currentAudioId = "0";
+    element.audioTracks = [
+      {id: "0", kind: "main", label: "eng", language: "eng", enabled: true},
+      {id: "1", kind: "main", label: "ger", language: "ger", enabled: false}
+    ];
+    var returns = wrapper.setVideoUrl("1");
+    expect(returns).to.be(true);
+  });
```

Comment:
Can we add a unit test to check that passing in an invalid track id does not cause any issues?